manifests/service_limits.pp

```diff
@@ -75,7 +75,6 @@
       path        => $::path,
       refreshonly => true,
       subscribe   => File["${path}/${name}.d/90-limits.conf"],
-      require     => Class['systemd::systemctl::daemon_reload'],
```

Comment:
I'm a bit unsure about this (which is also why this is a draft). This isn't a service type so it doesn't perform the reload if needed. Not sure how to deal with this.

Reply:
@raphink any thoughts on this?

Reply:
Looks likes it should stay then...

Reply:
But if the class to require is gone, should there be a local daemon reload?